applications/luci-app-firewall/htdocs/luci-static/resources/tools/firewall.js

```diff
@@ -496,6 +497,14 @@ return baseclass.extend({
 					ip4addrs[i].address, ' (', E('strong', {}, [dev]), ')'
 				]));
 			}
+			for (var i = 0; i < ip6addrs.length; i++) {
```

Comment:
The IPv6 address suggestions should only be added if `L.hasSystemFeature('firewall4')`